tests/test_slac_utils.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Test cases for utils."""
+import unittest
+import lsst.utils.tests
+
+from lsst.rubintv.production.slac.utils import getGains
+
+
+class RubinTVSlacUtilsTestCase(lsst.utils.tests.TestCase):
+    """A test case RubinTV SLAC utility functions."""
+
+    def test_getGains(self):
+        # check we have all the gains for TS8
+        ts8Gains = getGains('LSST-TS8')
+        self.assertEqual(len(ts8Gains), 9)
+        for ccd, gains in ts8Gains.items():
+            self.assertEqual(len(gains), 16)
+
+        # check we have all the gains for ComCam
+        comCamGains = getGains('LSSTComCam')
+        self.assertEqual(len(comCamGains), 9)
+        for ccd, gains in comCamGains.items():
+            self.assertEqual(len(gains), 16)
+
```

Comment:
I find it somewhat weird that the unit tests are verifying the data, but otherwise okay.

Reply:
In principle, I totally agree! The reason I put this in though, was just that up until now I was applying the TS8 gains to all instruments (albeit basically deliberately, as it was only coded up for TS,8 and then I used the same code for the other instruments) so I just added this to confirm that I wasn't still accidentally serving the same sets of gains to all instruments.